qiita_pet/templates/study_description.html

```diff
@@ -580,4 +601,21 @@
   {% end %}
 </div>
 
+
+<div class="modal fade delete-study" tabindex="-1" role="dialog" id="delete-study">
+  <div class="modal-dialog modal-sm">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h3>Deleting:<br/></h3><h4>{{study_title}}</h4>
+        You will only be able to delete a study that has no data associated with it.
+      </div>
+      <div class="modal-body">
```

Comment:
And if you follow the comment above, this should be moved to the "modal-footer" section.